### PR TITLE
New version: JackMordaunt.icnsify version 2.2.6

### PR DESCRIPTION
**Pull request by JackMordaunt**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/105831)

### CODE REVIEW

**Comment by JackMordaunt** [comment]
What's wrong with having a liberapay link in the readme?

**Comment by sitiom** [comment]
> What's wrong with having a liberapay link in the readme?

The Liberapay link came from your GitHub profile:
image

The validation is throwing an error on that link for some reason.

**Comment by Trenly** [comment]
> What's wrong with having a liberapay link in the readme?

The validation pipeline ensures all the URLs are reachable. For some reason, it wasn’t able to reach this URL. Specifically, when it tried to connect the liberapay site responded with a [403 Status Code](https://www.rfc-editor.org/rfc/rfc9110.html#name-403-forbidden). 

Since the pipelines can’t validate that it is a working URL, it disallows the URL

File: manifests/j/JackMordaunt/icnsify/2.2.6/JackMordaunt.icnsify.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Created with WinGet Releaser v2 using Komac v1.5.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JackMordaunt.icnsify
+PackageVersion: 2.2.6
+PackageLocale: en-US
+Publisher: Jack Mordaunt
+PublisherUrl: https://liberapay.com/JackMordaunt
```
Comment (Trenly) [inline manifests/j/JackMordaunt/icnsify/2.2.6/JackMordaunt.icnsify.locale.en-US.yaml:8]:
```suggestion
# PublisherUrl: https://liberapay.com/JackMordaunt
```

Comment (stephengillie) [reply manifests/j/JackMordaunt/icnsify/2.2.6/JackMordaunt.icnsify.locale.en-US.yaml:8]: Why should this be commented?

**Comment by stephengillie** [comment]
This package is expecting arguments:
```
Usage of C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\JackMordaunt.icnsify__DefaultSource\icnsify.exe:
  -i, --input string    Input image for conversion to icns from jpg|png or visa versa.
  -o, --output string   Output path, defaults to <path/to/image>.(icns|png) depending on input.
  -r, --resize int      Quality of resize algorithm. Values range from 0 to 5, fastest to slowest execution time. Defaults to slowest for best quality. (default 5)

You can also pipe to stdin and from stdout.
The pipes will be detected automatically, and both --input and --output will be ignored.

                cat icon.png | icnsify > icon.icns
```

File: manifests/j/JackMordaunt/icnsify/2.2.6/JackMordaunt.icnsify.locale.en-US.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/j/JackMordaunt/icnsify/2.2.6/JackMordaunt.icnsify.locale.en-US.yaml:8]: It's just one way to get past the 403 status code causing the URL Error

**Comment by sitiom** [comment]
Continued in #107360

**Pull request closed without merging**